packages/toggle/src/item.tsx

```diff
@@ -26,19 +27,35 @@ export function Item({
   invalid,
   value,
   helpId,
+  indeterminate,
   checked,
   defaultChecked,
   noVisibleLabel,
   labelClassName,
   ...props
 }: ItemProps) {
   const id = useId();
+  const checkboxRef = React.useRef<HTMLInputElement | null>(null);
 
   const labelContent = !children ? label || option?.label : children;
 
+  React.useEffect(() => {
+    if (!checkboxRef.current) {
+      return;
+    }
+
+    // 'indeterminate' state of checkbox cannot be assigned via HTML: https://developer.mozilla.org/en-US/docs/Web/HTML/Element/input/checkbox#indeterminate_state_checkboxes
+    if (indeterminate) {
+      checkboxRef.current.indeterminate = true;
+    } else {
+      checkboxRef.current.indeterminate = false;
+    }
```

Comment:
This could be simplified with the below

```suggestion
checkboxRef.current.indeterminate = indeterminate;
```

Reply:
Fixed in [7cd0b0c](https://github.com/fabric-ds/react/pull/161/commits/7cd0b0c97c6ab7513b3735bbe02cc15ffe24a73b)